### DISCUSSION TITLE
Attack without entity

### DISCUSSION THREAD

**Discussion by hexatester** [PrismarineJS/mineflayer#1683] [Q&A]
Is there any method to simulate left click without attacking entity?
To activate RPG item.

**Comment by ghost** [comment]
> Is there any method to simulate left click without attacking entity?
> To activate RPG item.

Use `bot.activateItem()`

**Reply by hexatester** [reply]
Doesn't it right click the item?

**Reply by ghost** [reply]
> Doesn't it right click the item?

`bot.activateItem(offHand=false)`  for the main hand and `bot.activateItem(offHand=true)` for the second.
Or do you need to click on an item in the window? Can you write more about the question?

**Reply by amoraschi** [reply]
I think he means bot.activateEntity

**Comment by miranda1000** [comment]
I'm also looking into this. The previous proposed solution (`bot.activateItem()`) I think it's the equivalent of **right** click (use), and we need the left click (attack).
I've tried `bot.swingArm()` but I think it doesn't do the trick.
